CONTRIBUTING.md

```diff
@@ -62,9 +69,18 @@ This repo is a “monorepo,” meaning it contains several projects in one. It c
    pnpm i
    ```
 
-#### Developing using Gitpod
+#### Developing using GitHub Codespaces
+
+To get started, create a codespace for this repository by clicking this 👇
 
-Instead of working locally on your machine, you can also contribute using an online coding development environment like Gitpod.
+[![Open in GitHub Codespaces](https://github.com/codespaces/badge.svg)](https://codespaces.new/withastro/starlight)
+
+Your new codespace will open in a web-based version of Visual Studio Code. All development dependencies will be preinstalled.
+
+> **Note**
+> Dev containers is now an open spec which is supported by [GitHub Codespaces](https://github.com/codespaces) and [other supporting tools](https://containers.dev/supporting).
+
```

Comment:
I think we can remove this as it’s not really relevant to someone wanting to contribute.

```suggestion
```

Reply:
Except providing a hint that the same devcontainer can be used in other ways, like VSCode with Docker Desktop (how I work).

Also based on https://github.com/withastro/astro/blob/main/CONTRIBUTING.md#using-github-codespaces-for-development